internal/internal_task_handlers_test.go

```diff
@@ -416,7 +417,7 @@ func (t *TaskHandlersTestSuite) TestWorkflowTask_ActivityTaskScheduled() {
 	t.NotNil(response.Decisions[0].CompleteWorkflowExecutionDecisionAttributes)
 }
 
-func (t *TaskHandlersTestSuite) TestWorkflowTask_QueryWorkflow_Sticky() {
+func (t *TaskHandlersTestSuite) TestWorkflowTask_QueryWorkflow() {
 	// Schedule an activity and see if we complete workflow.
 	taskList := "sticky-tl"
```

Comment:
"sticky-tl" does nothing special whatsoever, which is the same as now nonsticky uses "tl" which does nothing special whatsoever.

I'm not sure what these tests were doing when they were written, but it feels like bad copypasta.